src/System.Text.Json/tests/Serialization/Stream.ReadTests.cs

```diff
@@ -79,5 +81,17 @@ public static async Task ReadPrimitivesWithTrailingTriviaAsync()
                 Assert.Equal(1, i);
             }
         }
+
+        [Fact]
+        public static async Task ReadCollectionPassingNullValue()
```

Comment:
nit: I know we aren't consistent (at least in tests), but generally async methods have `Async` in the method name, as a suffix.

```suggestion
        public static async Task ReadCollectionPassingNullValueAsync()
```